packages/clay-button/src/__tests__/__snapshots__/ClayButton.js.snap

```diff
@@ -4,6 +4,8 @@ exports[`ClayButton should generate the default markup 1`] = `<button class="btn
 
 exports[`ClayButton should render a block button 1`] = `<button class="btn btn-block btn-primary"></button>`;
 
+exports[`ClayButton should render a button with custom classes 1`] = `<button class="btn my-custom-class btn-primary"></button>`;
```

Comment:
Hey @carloslancha wouldn't it be better to just leave `my-custom-class`? If I use this along with the badge, it may break because of the `btn btn-primary`, do not you think?